spec/factories/participants.rb

```diff
@@ -4,4 +4,14 @@
     sequence :number
     bus
   end
+
+  trait :total_score do
```

Comment:
This is supposed to be for the new pull request. I will have it commented it out for now. However, I have changed its name to better reflect it's function as 'initialize_total_score' because it will initialize the attributes necessary to have a valid total score.